### DISCUSSION TITLE
Corrupt PCAP Files After Upgrade

### DISCUSSION THREAD

**Discussion by travisanderson65** [Security-Onion-Solutions/securityonion#6698] [Unsupported Versions] [answered]
Hello I have installed SO for a few weeks now and I was working properly. After a soup update today where new docker containers were pulled and a full update done I was not able to access the PCAP files. I do not know what is wrong I even ran the so-tcpreplay too get back the image but that did not work. I really need some help if anyone could assist. 

![image](https://user-images.githubusercontent.com/37345585/147417854-7d9fd223-76af-4c28-bc3b-2ec461e68620.png)

**Comment by dougburks** [comment, marked as answer]
What is the output of the following?
```
df -h
```
```
sudo docker images
```
```
sudo so-status
```

**Reply by travisanderson65** [reply]
The output for df -h: 
"Filesystem               Size  Used Avail Use% Mounted on
devtmpfs                  12G     0   12G   0% /dev
tmpfs                     12G  320K   12G   1% /dev/shm
tmpfs                     12G   44M   12G   1% /run
tmpfs                     12G     0   12G   0% /sys/fs/cgroup
/dev/mapper/system-root   82G   37G   45G  46% /
/dev/sda2                497M  192M  305M  39% /boot
/dev/mapper/system-nsm   159G  141G   18G  89% /nsm
/dev/mapper/system-tmp   2.0G   62M  1.9G   4% /tmp
tmpfs                    2.4G   12K  2.4G   1% /run/user/42
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/35e62d52f1be43a32b99a9eccc5710aa2626c8dfdf0125c2bf2a28e711dbc06f/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/843a78d976c1e6ec7e489bd76a76d4c766a081fcb4f5bea27ef3152f4301d325/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/6483e76c3ff6a4a2020f275c7b784871b12e81ad97b0edf4a29dc5e060c06ddb/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/4ca1c4171df729da80b5b6bda80ae11c6dd23e27f4539c0e1fafaf8fe713bcc4/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/f185ae637a0633a8f088d18d82d7824424aab7411c06809b1d2ff3656421bdda/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/9c745922257fa346a3a08ba564a0898ae827c79cc17babc2d2eab9459cf21bf0/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/5c2b99fc12e1b727eea6858b84ac8b4435039779259cc5d2251edab38e386180/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/c4939ff9a32e4562e6d8dd4a3582bf0f2f0b8c6e7fa7de3f054451bd78610576/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/db3b84e3298d0c98301d6d7984c40807f1ae5c97f004fe98cb6a7cab0d977028/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/16b5ba56c1b3f1004288637a21917a7818b600bb25ac0de87fe29773ca5208f5/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/565bef6a6e510d750a50352987f86caef3d13d614c10e2941769b508817ff030/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/fa6a41634b0dcfd89dbfe9ae65fa23776e3638c2f0f9e896af0e360b8bc4c406/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/e901c08c2830542d50a93db788f5f7ed32ffe13dcff6386ea5d24fcab3ee3686/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/df0f589d20d401700f11dd9be2d3a32dbafc0670874c74bf7bb944c749552e04/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/eb15ae3fa93ebbacc3d909c7614457dd7e83d6f415a2bf446f96b6081584de84/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/c99d0237751740fc360bc5fe584aece947f68200d3f910b2f25b36bc11caf958/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/23c9f16946eac75086df53676403b7481e8a3ebb69ef901e61f34e87b829699b/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/802de84d2194c04d00771d8b1c4a7df7679a3a52379e04421eace67ce8d6afb5/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/411aec509f49e890d8ca6a86961b632a762780b0e33c73f60a69d4efff71992b/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/a54babc1abd54f7e541e96adcdf76e0a6176ebe12ec6e7b609409a88da43d255/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/cf94ef7bfb57eefb27c0c4e1f5a01256908017fb239b30d12e20da90650f7806/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/8edea7f6af5b890c02076547015d5a033e866bc33d4d20f3836457f2b3549f98/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/52f2e15b77cc604934c4e9e0d007fa03e8ba11268fd7f23af94f05aebe3c9adc/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/4fdd1dd811263717898b8b971fb96a7bdffa8ee0a8894101769fb311b4c4d075/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/47998fe033a992760cf7d40c802a1a9bd15cd16d86b6bbd77cf8afff91791997/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/151636e89bdac982bc86e580b27308e34a01877a8c843814af7e02847a2d360a/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/896dda4bd5c03468a06cf007fac802bd28fa07bf2afacc57ca1ec7bf1cb478ec/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/1e903bb184fbbf9d42ceb763889109dc7bb2e68e8070546be0d7df7e0fa1e9ce/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/e39e13f323e19febbabcacd5f017aca6ea311ba851155f9de9e6b8931d656758/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/1c19baf08457ddb5a847ce0c350e9c8292a6b75e4014619c62360192051dea8b/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/a84b8a0b6ee493633ba83bc0ae1a418994b5074969c5e2ee8a701125a590d9a9/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/157383b69154081dec1125316d96db3e233f0763d7e1defb854383816a78b82e/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/0d9f8438c9583897a31eb7536b7297d268d39fd3b310fbb33fe74d8de5296f82/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/1c201b4c06d57404569c2d7082d416f4094a2c1ee24578652b2c02130ab2e0a6/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/abff5eed1e0e2a966ff20c2e2d35307d068039c097911f3bda5db5b2bb23cdf5/merged
overlay                   82G   37G   45G  46% /var/lib/docker/overlay2/5515cf50fef9d929a4debf1e309bae5d1784d35bb742e1d144daddfe807c6ebe/merged
tmpfs                    2.4G     0  2.4G   0% /run/user/1000
tmpfs                    2.4G     0  2.4G   0% /run/user/0
"
Docker Images: 
"REPOSITORY                                               TAG       IMAGE ID       CREATED        SIZE
ghcr.io/security-onion-solutions/so-soc                  2.3.91    594b57216c74   6 days ago     262MB
so:5000/security-onion-solutions/so-soc                  2.3.91    594b57216c74   6 days ago     262MB
so:5000/security-onion-solutions/so-suricata             2.3.91    28f089fe73a4   7 days ago     746MB
ghcr.io/security-onion-solutions/so-suricata             2.3.91    28f089fe73a4   7 days ago     746MB
ghcr.io/security-onion-solutions/so-kratos               2.3.91    b2a032c302a7   7 days ago     56.6MB
so:5000/security-onion-solutions/so-kratos               2.3.91    b2a032c302a7   7 days ago     56.6MB
so:5000/security-onion-solutions/so-elasticsearch        2.3.91    34e3165ff09c   7 days ago     647MB
ghcr.io/security-onion-solutions/so-elasticsearch        2.3.91    34e3165ff09c   7 days ago     647MB
ghcr.io/security-onion-solutions/so-wazuh                2.3.91    9b923b214fb9   7 days ago     885MB
so:5000/security-onion-solutions/so-wazuh                2.3.91    9b923b214fb9   7 days ago     885MB
ghcr.io/security-onion-solutions/so-thehive-es           2.3.91    7ce2964461cc   7 days ago     1.2GB
so:5000/security-onion-solutions/so-thehive-es           2.3.91    7ce2964461cc   7 days ago     1.2GB
ghcr.io/security-onion-solutions/so-zeek                 2.3.91    9eb93f9fabff   7 days ago     599MB
so:5000/security-onion-solutions/so-zeek                 2.3.91    9eb93f9fabff   7 days ago     599MB
ghcr.io/security-onion-solutions/so-tcpreplay            2.3.91    7d0d7e196be2   7 days ago     912MB
so:5000/security-onion-solutions/so-tcpreplay            2.3.91    7d0d7e196be2   7 days ago     912MB
ghcr.io/security-onion-solutions/so-thehive-cortex       2.3.91    7a79219d4a5b   7 days ago     1.75GB
so:5000/security-onion-solutions/so-thehive-cortex       2.3.91    7a79219d4a5b   7 days ago     1.75GB
ghcr.io/security-onion-solutions/so-logstash             2.3.91    6fe6a31f3ee6   7 days ago     1.67GB
so:5000/security-onion-solutions/so-logstash             2.3.91    6fe6a31f3ee6   7 days ago     1.67GB
ghcr.io/security-onion-solutions/so-grafana              2.3.91    14359bb77b78   7 days ago     546MB
so:5000/security-onion-solutions/so-grafana              2.3.91    14359bb77b78   7 days ago     546MB
ghcr.io/security-onion-solutions/so-strelka-frontend     2.3.91    5f0d20504932   7 days ago     21.8MB
so:5000/security-onion-solutions/so-strelka-frontend     2.3.91    5f0d20504932   7 days ago     21.8MB
ghcr.io/security-onion-solutions/so-kibana               2.3.91    ed58e46a8813   7 days ago     2.06GB
so:5000/security-onion-solutions/so-kibana               2.3.91    ed58e46a8813   7 days ago     2.06GB
ghcr.io/security-onion-solutions/so-idstools             2.3.91    8ac5c9c20124   7 days ago     61.7MB
so:5000/security-onion-solutions/so-idstools             2.3.91    8ac5c9c20124   7 days ago     61.7MB
ghcr.io/security-onion-solutions/so-steno                2.3.91    36fcc3d9e329   7 days ago     850MB
so:5000/security-onion-solutions/so-steno                2.3.91    36fcc3d9e329   7 days ago     850MB
ghcr.io/security-onion-solutions/so-thehive              2.3.91    dd8aadc27cfc   7 days ago     838MB
so:5000/security-onion-solutions/so-thehive              2.3.91    dd8aadc27cfc   7 days ago     838MB
ghcr.io/security-onion-solutions/so-strelka-backend      2.3.91    ccdba8b509e6   7 days ago     1.07GB
so:5000/security-onion-solutions/so-strelka-backend      2.3.91    ccdba8b509e6   7 days ago     1.07GB
ghcr.io/security-onion-solutions/so-playbook             2.3.91    04c6d74446a0   7 days ago     605MB
so:5000/security-onion-solutions/so-playbook             2.3.91    04c6d74446a0   7 days ago     605MB
ghcr.io/security-onion-solutions/so-strelka-filestream   2.3.91    7e948d4ee004   7 days ago     20.6MB
so:5000/security-onion-solutions/so-strelka-filestream   2.3.91    7e948d4ee004   7 days ago     20.6MB
so:5000/security-onion-solutions/so-strelka-manager      2.3.91    0bd676ee36e5   7 days ago     21.6MB
ghcr.io/security-onion-solutions/so-strelka-manager      2.3.91    0bd676ee36e5   7 days ago     21.6MB
ghcr.io/security-onion-solutions/so-fleet-launcher       2.3.91    ebd967a52866   7 days ago     584MB
so:5000/security-onion-solutions/so-fleet-launcher       2.3.91    ebd967a52866   7 days ago     584MB
ghcr.io/security-onion-solutions/so-influxdb             2.3.91    7f1b1b59c767   7 days ago     183MB
so:5000/security-onion-solutions/so-influxdb             2.3.91    7f1b1b59c767   7 days ago     183MB
so:5000/security-onion-solutions/so-nginx                2.3.91    58136fc46d4a   7 days ago     86.5MB
ghcr.io/security-onion-solutions/so-nginx                2.3.91    58136fc46d4a   7 days ago     86.5MB
ghcr.io/security-onion-solutions/so-mysql                2.3.91    e14e49010d4f   7 days ago     224MB
so:5000/security-onion-solutions/so-mysql                2.3.91    e14e49010d4f   7 days ago     224MB
ghcr.io/security-onion-solutions/so-curator              2.3.91    a4b55b4aacc9   7 days ago     159MB
so:5000/security-onion-solutions/so-curator              2.3.91    a4b55b4aacc9   7 days ago     159MB
ghcr.io/security-onion-solutions/so-soctopus             2.3.91    d507db11e331   7 days ago     518MB
so:5000/security-onion-solutions/so-soctopus             2.3.91    d507db11e331   7 days ago     518MB
ghcr.io/security-onion-solutions/so-pcaptools            2.3.91    175d8c450b2c   7 days ago     223MB
so:5000/security-onion-solutions/so-pcaptools            2.3.91    175d8c450b2c   7 days ago     223MB
ghcr.io/security-onion-solutions/so-telegraf             2.3.91    0973ee95c40e   7 days ago     153MB
so:5000/security-onion-solutions/so-telegraf             2.3.91    0973ee95c40e   7 days ago     153MB
ghcr.io/security-onion-solutions/so-elastalert           2.3.91    572d62b8be2e   7 days ago     307MB
so:5000/security-onion-solutions/so-elastalert           2.3.91    572d62b8be2e   7 days ago     307MB
ghcr.io/security-onion-solutions/so-redis                2.3.91    94b4624dcce5   7 days ago     32.3MB
so:5000/security-onion-solutions/so-redis                2.3.91    94b4624dcce5   7 days ago     32.3MB
ghcr.io/security-onion-solutions/so-filebeat             2.3.91    cda90dc2f821   7 days ago     652MB
so:5000/security-onion-solutions/so-filebeat             2.3.91    cda90dc2f821   7 days ago     652MB
ghcr.io/security-onion-solutions/so-acng                 2.3.91    e8c6fad396a2   7 days ago     81.9MB
so:5000/security-onion-solutions/so-acng                 2.3.91    e8c6fad396a2   7 days ago     81.9MB
ghcr.io/security-onion-solutions/so-fleet                2.3.91    e570848ffdc8   7 days ago     93.9MB
so:5000/security-onion-solutions/so-fleet                2.3.91    e570848ffdc8   7 days ago     93.9MB
so:5000/security-onion-solutions/so-soc                  2.3.90    244f84b2bfba   5 weeks ago    256MB
so:5000/security-onion-solutions/so-suricata             2.3.90    653e8549cf2f   5 weeks ago    736MB
so:5000/security-onion-solutions/so-fleet                2.3.90    9c82dda16ce7   6 weeks ago    93.9MB
so:5000/security-onion-solutions/so-mysql                2.3.90    cc937a11f9ed   6 weeks ago    224MB
so:5000/security-onion-solutions/so-zeek                 2.3.90    c69d7ea2810e   6 weeks ago    596MB
so:5000/security-onion-solutions/so-nginx                2.3.90    d0061ee59bb6   6 weeks ago    86.5MB
so:5000/security-onion-solutions/so-playbook             2.3.90    239ee66166a8   6 weeks ago    604MB
so:5000/security-onion-solutions/so-thehive-cortex       2.3.90    87558a4e9de6   6 weeks ago    1.77GB
so:5000/security-onion-solutions/so-kibana               2.3.90    02f8ec58a28f   6 weeks ago    1.98GB
so:5000/security-onion-solutions/so-thehive-es           2.3.90    1c2b3fabd557   6 weeks ago    1.34GB
so:5000/security-onion-solutions/so-steno                2.3.90    4b82efe65fa1   6 weeks ago    849MB
so:5000/security-onion-solutions/so-strelka-manager      2.3.90    81fb55534917   6 weeks ago    21.6MB
so:5000/security-onion-solutions/so-elastalert           2.3.90    89e02d76f475   6 weeks ago    307MB
ghcr.io/security-onion-solutions/so-tcpreplay            2.3.90    1fea4af0f9dc   6 weeks ago    904MB
so:5000/security-onion-solutions/so-tcpreplay            2.3.90    1fea4af0f9dc   6 weeks ago    904MB
so:5000/security-onion-solutions/so-elasticsearch        2.3.90    af0756fb28af   6 weeks ago    792MB
so:5000/security-onion-solutions/so-curator              2.3.90    4d073ea3689c   6 weeks ago    155MB
so:5000/security-onion-solutions/so-strelka-backend      2.3.90    1d9f8d88a51b   6 weeks ago    1.07GB
so:5000/security-onion-solutions/so-kratos               2.3.90    41ea23977f2f   6 weeks ago    56.6MB
so:5000/security-onion-solutions/so-influxdb             2.3.90    4aaacb8bb64d   6 weeks ago    183MB
so:5000/security-onion-solutions/so-idstools             2.3.90    620cc8b0a318   6 weeks ago    61.7MB
so:5000/security-onion-solutions/so-redis                2.3.90    891a2ceaad96   6 weeks ago    32.3MB
so:5000/security-onion-solutions/so-wazuh                2.3.90    d030ed033912   6 weeks ago    884MB
so:5000/security-onion-solutions/so-logstash             2.3.90    134e14ca192a   6 weeks ago    1.69GB
so:5000/security-onion-solutions/so-strelka-filestream   2.3.90    a6e7c0bd6153   6 weeks ago    20.6MB
so:5000/security-onion-solutions/so-thehive              2.3.90    baf04d2a0b8f   6 weeks ago    838MB
so:5000/security-onion-solutions/so-filebeat             2.3.90    d824695284f7   6 weeks ago    654MB
so:5000/security-onion-solutions/so-soctopus             2.3.90    d87d6e51b249   6 weeks ago    517MB
so:5000/security-onion-solutions/so-grafana              2.3.90    2b5ba919c9d8   6 weeks ago    546MB
so:5000/security-onion-solutions/so-strelka-frontend     2.3.90    a729c73f2ea7   6 weeks ago    21.8MB
so:5000/security-onion-solutions/so-fleet-launcher       2.3.90    4626bfe16782   6 weeks ago    584MB
so:5000/security-onion-solutions/so-acng                 2.3.90    a45cd0c00d74   6 weeks ago    81.9MB
so:5000/security-onion-solutions/so-telegraf             2.3.90    bc41e3e6442c   6 weeks ago    153MB
ghcr.io/security-onion-solutions/registry                latest    b2cb11db9d3d   3 months ago   26.2MB
wlambert/so-fir                                          latest    0d4c6eeada16   3 years ago    297MB
"
So-status: 
"
Checking Docker status

    Docker -------------------------------------------------------------------------------------------------------------------- [ OK ]

Checking container statuses

    so-aptcacherng ------------------------------------------------------------------------------------------------------------ [ OK ]
    so-cortex ----------------------------------------------------------------------------------------------------------------- [ OK ]
    so-curator ---------------------------------------------------------------------------------------------------------------- [ OK ]
    so-dockerregistry --------------------------------------------------------------------------------------------------------- [ OK ]
    so-elastalert ------------------------------------------------------------------------------------------------------------- [ OK ]
    so-elasticsearch ---------------------------------------------------------------------------------------------------------- [ OK ]
    so-filebeat --------------------------------------------------------------------------------------------------------------- [ OK ]
    so-fleet ------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-grafana ---------------------------------------------------------------------------------------------------------------- [ OK ]
    so-idstools --------------------------------------------------------------------------------------------------------------- [ OK ]
    so-influxdb --------------------------------------------------------------------------------------------------------------- [ OK ]
    so-kibana ----------------------------------------------------------------------------------------------------------------- [ OK ]
    so-kratos ----------------------------------------------------------------------------------------------------------------- [ OK ]
    so-logstash --------------------------------------------------------------------------------------------------------------- [ OK ]
    so-mysql ------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-nginx ------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-playbook --------------------------------------------------------------------------------------------------------------- [ OK ]
    so-redis ------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-sensoroni -------------------------------------------------------------------------------------------------------------- [ OK ]
    so-soc -------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-soctopus --------------------------------------------------------------------------------------------------------------- [ OK ]
    so-steno ------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-strelka-backend -------------------------------------------------------------------------------------------------------- [ OK ]
    so-strelka-coordinator ---------------------------------------------------------------------------------------------------- [ OK ]
    so-strelka-filestream ----------------------------------------------------------------------------------------------------- [ OK ]
    so-strelka-frontend ------------------------------------------------------------------------------------------------------- [ OK ]
    so-strelka-gatekeeper ----------------------------------------------------------------------------------------------------- [ OK ]
    so-strelka-manager -------------------------------------------------------------------------------------------------------- [ OK ]
    so-suricata --------------------------------------------------------------------------------------------------------------- [ OK ]
    so-telegraf --------------------------------------------------------------------------------------------------------------- [ OK ]
    so-thehive ---------------------------------------------------------------------------------------------------------------- [ OK ]
    so-thehive-es ------------------------------------------------------------------------------------------------------------- [ OK ]
    so-wazuh ------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-zeek ------------------------------------------------------------------------------------------------------------------- [ OK ]
"

**Reply by dougburks** [reply]
When your disk starts reaching capacity, it has to purge old pcap files and this results in the message that you're seeing.

Looks like you have about 240GB of disk space. This is barely above the minimum requirements:
https://docs.securityonion.net/en/2.3/hardware.html#minimum-specs

You'll likely want much more than 240GB for a true production deployment.

**Comment by travisanderson65** [comment]
Before the upgrade the VM has 500GB of storage. How do I allocate it to the server On Dec 28, 2021 1:46 PM, Doug Burks ***@***.***> wrote:
When your disk starts reaching capacity, it has to purge old pcap files and this results in the message that you're seeing.
Looks like you have about 240GB of disk space. This is barely above the minimum requirements:
https://docs.securityonion.net/en/2.3/hardware.html#minimum-specs
You'll likely want much more than 240GB for a true production deployment.

**Reply by dougburks** [reply]
> Before the upgrade the VM has 500GB of storage. How do I allocate it to the server

I'm not sure I understand. Your `df -h` output shows:
```
/dev/mapper/system-root 82G 37G 45G 46% /
/dev/sda2 497M 192M 305M 39% /boot
/dev/mapper/system-nsm 159G 141G 18G 89% /nsm
```
which totals to about 240GB. Where is the other 260GB? If it's assigned to the VM but not allocated to a partition, you could try something like https://docs.securityonion.net/en/2.3/new-disk.html. However, keep in mind that 500GB is still not much storage for a true production deployment where you are storing full packet capture:
https://docs.securityonion.net/en/2.3/hardware.html#id1

**Reply by travisanderson65** [reply]
Thank you so much @dougburks I really appreciate it. We wanted to implement the SO in our environment and we were just testing out what will be required for our current set up.
